src/Modal/PopperElement.test.jsx

```diff
@@ -45,7 +45,7 @@ describe('<PopperElement />', () => {
       </PopperElement>
     ));
     const popperEl = wrapper.find('[data-test="someValue"]');
-    expect(usePopper).toHaveBeenCalledWith(targetRef, null, defaultPopperOptions);
+    expect(usePopper).toHaveBeenCalledWith(<div />, null, defaultPopperOptions);
```

Comment:
nit: would probably be better to write `targetRef.current` instead of `<div />` (so if someone changes contents of `targetRef` they wouldn't need to change it here too)

Reply:
Done